packages/p/poco/xmake.lua

```diff
@@ -21,7 +21,7 @@ package("poco")
     end
     add_configs("mysql", {description = "Enable mysql support.", default = false, type = "boolean"})
     add_configs("postgresql", {description = "Enable postgresql support.", default = false, type = "boolean"})
-    add_configs("odbc", {description = "Enable odbc support.", default = is_plat("windows"), type = "boolean"})
+    add_configs("odbc", {description = "Enable odbc support.", default = is_plat("windows") or not is_plat("mingw"), type = "boolean"})
```

Comment:
why? `or not`?

Reply:
Odbc support must on mingw platform

Reply:
why not `is_plat("windows", "mingw")`?

> is_plat("windows") or not is_plat("mingw")

`is_plat("windows")` It is a redundant condition, becasee windows always satisfy `or not is_plat("mingw")`